workflows/nallo.nf

```diff
@@ -101,9 +103,14 @@ workflow NALLO {
     // Channels from (optional) input samplesheets validated by schema
     ch_databases                = createReferenceChannelFromSamplesheet(params.snp_db, 'assets/schema_snp_db.json')
     ch_vep_plugin_files         = createReferenceChannelFromSamplesheet(params.vep_plugin_files, 'assets/schema_vep_plugin_files.json', Channel.value([]))
+    ch_hgnc_ids                 = createReferenceChannelFromSamplesheet(params.filter_variants_hgnc_ids, 'assets/schema_hgnc_ids.json')
+        .map { integer -> 'HGNC:' + integer[0].toString() } // Converts [ 256 ] to a string HGNC:256
```

Comment:
It's because HGNC IDs need to be filtered like `1234` in hg19 but like `HGNC:1234` in hg38. Scout, which we will use in Stockholm to outputs gene panels export them without `HGNC:`.

It would be possible to accept the HGNC prefix into the pipeline if you think that makes more sense, then we will have to do some pre-processing. Might actually be more correct if `HGNC:` is in the actual ID. 
<img width="203" alt="Screenshot 2024-11-11 at 13 53 31" src="https://github.com/user-attachments/assets/64fb0f79-512e-4434-a59f-94442e5cb682">

Reply:
Actually, seems like VEP can use both now after 113 update. I will remove this and update the input schema.